### DISCUSSION TITLE
Spell components

### DISCUSSION THREAD

**Discussion by bhex** [CleverRaven/Cataclysm-DDA#47065] [Ideas]
I understand some spells that are high level require components but for example Flamesword consuming the sword is bit over the top in my opinion and also intrestingly enough to use the Flamesword you need to put it first in your inventory i.e. scabbard.

My suggestion would be either to remove the component requirement for lower level spells completely or relax the "you need your hands free" for this spell.

Another option would be to have the component requirement removed when higher level spellcasting is acquired.

**Comment by actual-nh** [comment]
At least part of the thing with Flamesword is being worked on (turns out to be more complicated than it looks) - #45941.

**Comment by bhex** [comment]
I just went and edited the component requirement out, I think the idea itself is great but should be usable on all blades and not destroy the original item.

Considering storm hammer is much stronger spell and requires no components.

Further thinking led me to idea that they should be elemental weapon buffs instead of summoning a weapon since that itself is quite OP.

Talking about OP: Druid Bow -spell comes to mind.
